release.sh

```diff
@@ -26,6 +26,7 @@ perl -i -pe 's/\d+\.\d+\.\K(\d+)/$1+1/e' $version_file
 version=$(grep -m1 -Eo "[0-9]+\.[0-9]+\.[0-9]+" $version_file)
 echo "Version: $version"
 
+docker pull fnproject/dind:17.12
```

Comment:
It will always get the latest dind during release already, this is probably unecessary.

Reply:
+1 to remove, we have https://github.com/fnproject/fn/blob/master/Dockerfile#L9 -- this line risks going stale vs the Dockerfile and we'd eat the pull

Reply:
While staleness is a problem, are we guaranteed that the docker caches on the ci machines are cleared between runs?

If so, I've no problem with dropping this. On t'other hand, if it doesn't *and* if we shift to "rebuild these tools when changed" then we risk a change on a branch leaving a broken dind behind for main to consume.

Reply:
... rather than ask I'll check it :-)

Reply:
with circleci they make you pay to have caching, and we're not ponying up, so I'm pretty sure the caches are cleared out (it has been a bit of a thorn, to date)

Reply:
We can explicitly clear them in Makefile (eg. clear-images target) and hook that up to circleci/config.yaml.